platform/platform-configuration-listener/src/main/java/org/codice/ddf/configuration/store/ConfigurationFileDirectory.java

```diff
@@ -151,13 +164,13 @@ private void moveConfigurationFile(Path source, Path destination) {
         return configurationFiles;
     }
 
-    private Collection<Path> listFiles() throws IOException {
+    private Collection<Path> listFiles()throws IOException {
         Collection<Path> fileNames = new ArrayList<>();
 
-        for (Path path : directoryStream) {
+        for (Path path : configDirectoryStream) {
             fileNames.add(path);
         }
-
+        configDirectoryStream.close();
```

Comment:
This is going to change the behavior of this method so that it only returns a Collection<Path> once and then never again. Is that what you want? I ask, because that is a very strange pattern, if so. It'd make more sense to me to read the list of files in the constructor and then store the Collection<Path> as an instance field. Then you don't have the possibility of a resource leak, like you do here.